repos/system_upgrade/common/actors/systemd/checksystemdservicetasks/libraries/checksystemdservicetasks.py

```diff
@@ -0,0 +1,36 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import SystemdServicesTasks
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def _inhibit_upgrade_with_conflicts(conflicts):
+    summary = (
+        'The requested states for systemd services on the target system are in conflict.'
+        ' The following systemd services were requested to be both enabled and disabled on the target system: {}'
+    )
+    report = [
+        reporting.Title('Conflicting requirements of systemd service states'),
+        reporting.Summary(summary.format(_printable_conflicts(conflicts))),
+        reporting.Severity(reporting.Severity.HIGH),
+        reporting.Groups([reporting.Groups.SANITY]),
+        reporting.Groups([reporting.Groups.INHIBITOR]),
+    ]
+    reporting.create_report(report)
+
+
+def _printable_conflicts(conflicts):
+    return FMT_LIST_SEPARATOR + FMT_LIST_SEPARATOR.join(sorted(conflicts))
```

Comment:
Place the function please above `_inhibit_upgrade_with_conflicts`. It's just a good practice in this project to define function before their use. No strong feeling about that but even for review it's more friendly (at least for me). No blocker for the merge